cluster/ci/config/prow/openshift/jenkins-config-updater.yaml

```diff
@@ -0,0 +1,87 @@
+# Needs the hmac-token secret for decrypting webhook payloads.
+# Needs the oauth-token secret for creating PRs.
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: jenkins-config-updater-prow
+  annotations:
+    "openshift.io/display-name": jenkins-config-updater-prow
+  description: |
+    jenkins-config-updater manifests for prow
+parameters: []
+objects:
+- apiVersion: v1
+  kind: Route
```

Comment:
I don't think you need a route.

Reply:
For testing we did -- to not need to reconfigure `hook`